### PR TITLE
Improved and complete README

### PR DESCRIPTION
**Pull request by noelmace**
New README based on the [devkit README template](https://github.com/blindnet-io/openness-framework/blob/main/OpenSource/template/devkit-README.md).

see https://github.com/blindnet-io/devrel-management/issues/9 & https://github.com/blindnet-io/openness-framework/pull/33

:warning: @blindnet-io/engineering still has to add information to this README, and rewrite its content

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,50 +1,115 @@
...
+</p>
 
-```
-Java v11+
-Gradle v7.3.1+
-```
+## About
 
-# Building project
+{explain what this project does and why it exists in the most concise way possible.}
```
Comment (noelmace) [inline README.md:30]: TODO @blindnet-io/engineering

File: README.md [hunk 2]
```diff
@@ -1,50 +1,115 @@
...
-```
-gradle build
-```
+Those functionnalities are provided through two submodules:
 
-In order to run tests:
-```
-gradle clean test
+1. Core: {short explainer of what this module provide}
+2. Signal: {short explainer of what this module provide}
```
Comment (noelmace) [inline README.md:35]: TODO @blindnet-io/engineering

File: README.md [hunk 3]
```diff
@@ -1,50 +1,115 @@
...
+:rocket: Check out our [Quick Start Guide](https://blindnet.dev/docs/quickstart) to get started in a snap.
+
+## Installation
+
+This project requires [Java v11+](URL) {...} [Gradle v7.3.1+](URL) {...}
+
+Use [{package manager or plateform}][install-tool] to install {project's name}:
+
+```bash
+{install-command}
```
Comment (noelmace) [inline README.md:48]: TODO @blindnet-io/engineering

File: README.md [hunk 4]
```diff
@@ -1,50 +1,115 @@
...
 ```
 
-# Using SDK
+## Usage
+
+<!-- FIXME: RESTRUCTURE and REWRITE in plain and correct english, using clear code examples -->
+
+<!-- FIXME: publish the API reference -->
+
+> 📑 The API reference of blindnet devkit Java client SDK is available on [blindnet.dev](https://blindnet.dev/docs/api_reference/[path-to-project}/latest).
```
Comment (noelmace) [inline README.md:57]: TODO @blindnet-io/engineering

**Review by noelmace — COMMENTED** [review]
TODO List for @blindnet-io/engineering before merging this in order to close https://github.com/blindnet-io/devrel-management/issues/9

**Comment by veselinovicst** [comment]
@noelmace resolved all todo's

File: README.md (hunk 4 above)
Comment (noelmace) [reply README.md:77]: @veselinovicst you can comment this line, we'll address this in another issue and PR (see https://github.com/blindnet-io/sdk-java-server/pull/2#discussion_r929941909).

File: README.md [hunk 5]
```diff
@@ -42,10 +56,16 @@ Those functionnalities are provided through two submodules:
...
+
+```bash
+gradle build
+```
+
+In order to run tests use:
 
 ```bash
-{install-command}
+gradle clean test
```
Comment (noelmace) [inline README.md:68]: This section should explain how to use the SDK as an end user, not how to contribute to it.

File: README.md (hunk 4 above)
Comment (noelmace) [reply README.md:77]:
```suggestion
```

**Pull request merged**